sources/dialect/original_files/README

```diff
@@ -0,0 +1,582 @@
+Variant Conversion Info (VarCon)
```

Comment:
It's a Markdown or a text?

I would add a .md or .txt

Reply:
I would rather not change this one since that was the exact title from the text

Reply:
I'm unsure about what you said.

My point was to rename this README file to README.md to allow GitHub to render it correctly, but maybe it already works or is not necessary.

Feel free to check (no computer with me) and fix or resolve

Reply:
Oh, I see. Sorry, I was confused. I can't change the name of this document as this the original name for the document.